src/DssSpell.sol

```diff
@@ -78,9 +67,37 @@ contract DssSpell {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/aa1ccb0c0dea66c66a2fd7fba687b0b9e0478098/governance/votes/Executive%20vote%20-%20November%2020%2C%202020.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/a7c8ddb3f8d8ea71cb123b9aa45d9d7eaed8d6f0/governance/votes/Executive%20vote%20-%20November%2023%2C%202020.md -q -O - 2>/dev/null)"
     string constant public description =
-        "2020-11-20 MakerDAO Executive Spell | Hash: 0x132a0053c544cbc2772e17d62b7b1ddcfa0074cc8132597378cc9846168822f5";
+        "2020-11-23 MakerDAO Executive Spell | Hash: 0x3567e2282249022428233fe24a48a25ebc34468f2183869109f2bd590f48ef28";
+
+    // MIP24: Emergency Voting System
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/e9d0aad1e1fcfafbf21b3358dc778cfc3638a011/MIP24/mip24.md -q -O - 2>/dev/null)"
+    string constant public MIP24 = "0x6d39f78a3343fb030da792962abdd12ca1b0c9384b92f496e8a070e97cf3c1c6";
+
+    // MIP25: Flash Mint Module
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/e9d0aad1e1fcfafbf21b3358dc778cfc3638a011/MIP25/mip25.md -q -O - 2>/dev/null)"
+    string constant public MIP25 = "0xd2550d2b15464b6bf3e49bc424a85e6411abf27e72247c4325f6d9b2ba4d9100";
+
+    // MIP27: Debt Ceiling Instant Access Module
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/e9d0aad1e1fcfafbf21b3358dc778cfc3638a011/MIP27/mip27.md -q -O - 2>/dev/null)"
+    string constant public MIP27 = "0x2848c1ef785a2182d9ccd7171e90eba847330f3da2106500f0f3e097a3bf5553";
```

Comment:
In this file there is an extra line break at the end provoking that if you copy and paste it to https://emn178.github.io/online-tools/keccak_256.html the hash is different.

Reply:
Hmm what should we do?

Reply:
Yeah just checked it is different